treeherder/etl/perf.py

```diff
@@ -107,6 +106,7 @@ def _load_perf_datum(job, perf_datum):
                     'option_collection': option_collection,
                     'platform': job.machine_platform,
                     'extra_options': suite_extra_options,
+                    'measurement_unit': suite.get('unit'),
```

Comment:
Whenever a `performance_datum` belonging to existing `performance_signature` is ingested & has a different `unit`, the `performance_signature`'s measurement_unit also updates.
We need to provide test coverage for this.

Reply:
Ok, will do.